### PR TITLE
fix: MPR metadata parsing for spacings

### PR DESCRIPTION
**Pull request by sedghi**
MPR relies on floats for spacings, apparently new dcmjs update has crashed it

image

See 
- https://github.com/OHIF/Viewers/issues/2422
- https://github.com/OHIF/Viewers/issues/2318

### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: extensions/vtk/src/OHIFVTKViewport.js [hunk 1]
```diff
@@ -267,11 +267,13 @@ class OHIFVTKViewport extends Component {
     const spacing = vtkImageData.getSpacing();
     // Set the sample distance to half the mean length of one side. This is where the divide by 6 comes from.
     // https://github.com/Kitware/VTK/blob/6b559c65bb90614fb02eb6d1b9e3f0fca3fe4b0b/Rendering/VolumeOpenGL2/vtkSmartVolumeMapper.cxx#L344
-    const sampleDistance = (spacing[0] + spacing[1] + spacing[2]) / 6;
+    const sampleDistance =
+      (parseFloat(spacing[0]) + parseFloat(spacing[1]) + parseFloat(spacing[2])) /
```
Comment (igoroctaviano) [inline extensions/vtk/src/OHIFVTKViewport.js:271]: I think this should be a metadata provider change and not specifically to VTK?

**Review by igoroctaviano — CHANGES_REQUESTED** [review]
Left a comment

File: extensions/vtk/src/OHIFVTKViewport.js (hunk 1 above)
Comment (sedghi) [reply extensions/vtk/src/OHIFVTKViewport.js:271]: hmmm, it appears to be coming from here
https://github.com/OHIF/react-vtkjs-viewport/blob/HEAD/src/lib/data/buildMetadata.js#L3

Comment (igoroctaviano) [reply extensions/vtk/src/OHIFVTKViewport.js:271]: This function uses the pixel module, you just need to go to metadata provider in OHIF core and parse the data there (pixel spacing) and call it a day

Comment (sedghi) [reply extensions/vtk/src/OHIFVTKViewport.js:271]: VTK extension doesn't get its metadata from OHIF metadataProvider, it gets it from react-vtkjs-viewport

Comment (sedghi) [reply extensions/vtk/src/OHIFVTKViewport.js:271]: I'm looking at it right now

Comment (igoroctaviano) [reply extensions/vtk/src/OHIFVTKViewport.js:271]: If it's using a module from cornerstone then it's using the metadata provider which is fed by OHIF (given the DICOM files fetched) and passed to cornerstone

**Review by swederik — APPROVED** [review]

**Review by igoroctaviano — APPROVED** [review]
LGTM

**Pull request merged**